modules/ducktests/tests/ignitetest/tests/snapshot_test.py

```diff
@@ -63,8 +63,14 @@ def snapshot_test(self, ignite_version):
         loader = IgniteApplicationService(
```

Comment:
Can we invoke `preload_data` function here instead of manually running application

Reply:
Data loading code has been extracted to the separate class (DataLoader).  Both rebalance and snapshot tests use it now.